java/src/jmri/jmrit/logixng/implementation/DefaultCsvNamedTable.java

```diff
@@ -54,5 +62,13 @@ public String getFileName() {
     public void setFileName(String fileName) {
         this._fileName = fileName;
     }
-    
+
+    public void setCsvType(CsvType csvType) {
+        _csvType = csvType;
+    }
+
+    public CsvType getCsvType() {
+        return _csvType;
+    }
```

Comment:
I can move the enum to Table (and agree that is the better place for it), but the access to the CSV type needs to remain in DefaultCsvNamedTable because the csvtype  (like the filename) is only stored there. Should the other methods (get/set/is) be in the interface?

Reply:
It's answered in https://github.com/JMRI/JMRI/pull/11533#discussion_r1002715267

The idea with the method `isCsvTypeSupported()` is that the caller can check if Csv type is supported. If `isCsvTypeSupported()` returns false, the other two methods must throw an UnsupportedOperationException.

Reply:
figured it out - now done